### PR TITLE
feat(OHI PostgreSQL): Added branching doc structure

### PR DESCRIPTION
**Pull request by nbaenam**
Added the branching doc structure for this page: [PostgreSQL monitoring integration](https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/postgresql/postgresql-integration/).

This is the ticket: NR-189602

### CODE REVIEW

File: src/install/config/postgresql.yaml [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    placeholder: "Choose one:"
+    options:                      
+      - value: "apt"
+        displayName: "Install using Apt (Debian, Ubuntu)"
+        recommendedGuided: true        
+      - value: "yum"
+        logo: 'linux'      
+        displayName: "Install using Yum (Amazon Linux, CentOS, RHEL)"
+        recommendedGuided: true  
+      - value: "tar"
```
Comment (bradleycamacho) [inline src/install/config/postgresql.yaml:36]: I think this is missing the Zypper option

File: src/install/postgresql/default-requirements.mdx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+headingText: Check the compatibility and requirements
+componentType: default
+---
+
+
+* Our integration is compatible with PostgreSQL until version 16.
```
Comment (bradleycamacho) [inline src/install/postgresql/default-requirements.mdx:7]:
```suggestion
* Our integration is compatible with PostgreSQL version 15.6 and lower. 
```

Comment (bradleycamacho) [reply src/install/postgresql/default-requirements.mdx:7]: I always forget about this style guide rule! https://docs.newrelic.com/docs/style-guide/word-choice/usage-dictionary/#version

File: src/install/postgresql/find-data.mdx [hunk 3]
```diff
@@ -0,0 +1,26 @@
+---
+componentType: default
+headingText: Find and use data
+---
+
+
+import infrastructureInstallPostgresqlDashboard from 'images/infrastructure_screenshot-full_install-postgresql-dashboard.webp'
+
+
+1. Go to **[one.newrelic.com](https://one.newrelic.com) > Add data** and type **postgresql**.
```
Comment (bradleycamacho) [inline src/install/postgresql/find-data.mdx:10]:
```suggestion
1. Go to **[one.newrelic.com](https://one.newrelic.com) > Add data** and type **PostgreSQL**.
```

File: src/install/postgresql/linux/install-linux.mdx [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+        COLLECTION_LIST: 'ALL'
+        TIMEOUT:  10
+      interval: 15s
+      labels:
+        env: production
+        role: postgresql
+      inventory_source: config/postgresql
+  ```
+
+4. Before you restart the infrastructure agent, [create a user](#create-user) with `READ` permissions on the required functions.
```
Comment (bradleycamacho) [inline src/install/postgresql/linux/install-linux.mdx:40]: Should this be included in the Yum/Apt/Zypper files too?

File: src/install/postgresql/whatsNext.mdx [hunk 5]
```diff
@@ -0,0 +1,1182 @@
+---
+componentType: default
+headingText: What's next?
+---
+
+## PostgreSQL users and permissions [#create-user]
+
+Create a user with `SELECT` permissions on:
```
Comment (bradleycamacho) [inline src/install/postgresql/whatsNext.mdx:8]: I'm a bit confused about this section. It seems to be a mix of procedural and reference material. Also, some of the steps above tell people to create a user and link to this section.

I think this specific section needs a bit of a rewrite to pull out the procedural material from the reference material. The reference material should stay here, and the procedural material should be included as a separate step above.

File: src/install/postgresql/whatsNext.mdx [hunk 6]
```diff
@@ -0,0 +1,1182 @@
...
+GRANT pg_read_all_stats TO new_relic;
+```
+
+Enabling the `pg_stat_statements` extension may require you to manually create it from a query prompt:
+
+```sql
+CREATE EXTENSION pg_stat_statements;
+```
+
+### postgresql-config.yml sample files [#examples]
```
Comment (bradleycamacho) [inline src/install/postgresql/whatsNext.mdx:43]:
```suggestion
## postgresql-config.yml sample files [#examples]
```

**Review by bradleycamacho — CHANGES_REQUESTED** [review]

File: src/install/postgresql/default-requirements.mdx (hunk 2 above)
Comment (nbaenam) [reply src/install/postgresql/default-requirements.mdx:7]: Good catch!

File: src/install/postgresql/find-data.mdx (hunk 3 above)
Comment (nbaenam) [reply src/install/postgresql/find-data.mdx:10]: OK, I'm going to correct it, but, actually, it doesn't matter the way you write it.

File: src/install/postgresql/linux/install-linux.mdx (hunk 4 above)
Comment (nbaenam) [reply src/install/postgresql/linux/install-linux.mdx:40]: I've just the question to the help-core-integrations channel.

File: src/install/postgresql/whatsNext.mdx (hunk 6 above)
Comment (nbaenam) [reply src/install/postgresql/whatsNext.mdx:43]: Done!

**Pull request closed without merging**